notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/fcm/oktransport/OkGoogleHttpTransport.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.notifications.worker.delivery.fcm.oktransport
+
+import com.google.api.client.http.{HttpTransport, LowLevelHttpRequest}
+import okhttp3.{Dns, OkHttpClient}
+
+import scala.collection.JavaConverters._
+import scala.util.Random
+
+
+class OkGoogleHttpTransport extends HttpTransport {
+  private val okHttpClient = new OkHttpClient.Builder()
+    .dns((hostname: String) => Random.shuffle(Dns.SYSTEM.lookup(hostname).asScala).asJava)
+    .followRedirects(false)
```

Comment:
Shouldn't that be true by default?

Reply:
I thought so, but when I tested it against https://www.theguardian.com (which includes a redirect) the client FCM already uses does not redirect; so I disabled it to match the behaviour of the existing client.